roles/taca/templates/sthlm_fastq_taca_delivery.yml.j2

```diff
@@ -0,0 +1,21 @@
+log:
```

Comment:
Is this file only needed for Stockholm, or will there be one for Uppsala as well?